core/arch/arm/kernel/core_bootcfg.c

```diff
@@ -169,7 +169,7 @@ static struct map_area bootcfg_memory_map[] = {
 };
 
 /*
- * bootcfg_get_pbuf_is_handler - return the platform specfic pbuf_is
+ * bootcfg_get_pbuf_is_handler - return the platform specific pbuf_is
```

Comment:
My fault. *common* is too generic / vague.

I would like to change the file name in order to emphasize that core_bootcfg.c, entry_a32.S and entry_a64.S are only relevant when CFG_COMMON_BOOT=y